packages/cow-hooks-ui/src/hooks/cowShed/useHookDeadline.ts

```diff
@@ -7,13 +7,13 @@ export function useHookDeadline({
   context: HookDappContextAdjusted | undefined;
 }) {
   return useMemo(() => {
-    return BigInt(
-      context?.orderParams?.validTo || generateTimestampOnNextHour()
-    ); // TODO check valid to parameter
+    const currentTimestamp = new Date().getTime() / 1000;
+
+    if (
+      !context?.orderParams?.validTo ||
+      context?.orderParams?.validTo < currentTimestamp
+    )
+      return BigInt(currentTimestamp.toFixed() + 60 * 60);
```

Comment:
The issue was that we were getting some invalid `validTo` parameters. I checked and it seems that is because the timezone. I will fix it but I think that we should keep a min value for the deadline (1 hour ahead seems good IMO)